src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        int numberOfIteration = 3;
```

Comment:
it's not a constant, constant look like this `private static final BALLS_COUNT = 3;`